docs/content/3.middleware/2.guides/6.headers.md

```diff
@@ -0,0 +1,113 @@
+# Response Headers
+
+## Overview
+
+The Middleware Response Headers [extension](/middleware/guides/extensions) enables the addition of custom response headers, which can be particularly useful for managing application caching or establishing security rules. 
+
+With this extension, you can overwrite or set headers for specific API methods or apply them globally across your application. 
+It can be implemented for any Unified Alokai Backend integration that exposes API methods and allows the addition of custom headers.
```

Comment:
How does a developer know if this is the case? Is it documented in each integration? 

> and allows the addition of custom headers

Reply:
@rohrig All of the middleware allows.

Reply:
What's your suggestion? Should I mention that? Where are we heading here? :)